backend/polling.ml

```diff
@@ -278,7 +279,8 @@ let instrument_fundecl ~future_funcnames:_ (f : Mach.fundecl) : Mach.fundecl =
   end
 
 let requires_prologue_poll ~future_funcnames ~fun_name i =
-  if function_is_assumed_to_never_poll fun_name then false
+  if !Flambda_backend_flags.disable_poll_insertion ||
```

Comment:
(Maybe go through a utility function.)

Reply:
This seems ok I think, I'd like to get this in soon...